src/helpers/setup.js

```diff
@@ -8,9 +8,17 @@ const filterFlowBasedOnDevelopBranch = require("./filterFlowBasedOnDevelopBranch
 const utils = require("../utils");
 
 const setup = async options => {
+	options.scripts = await utils.getScripts(options.command);
```

Comment:
so why wasn't it work the way it was before?

Reply:
I changed the order again. I just needed to know about the scripts when it was checking the callback so I moved them up.